### ISSUE TITLE
"No preview available" message has insufficient colour contrast

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-cms#2671]
Within the CMS preview in Preview or Split mode you can sometimes get a message saying "No preview available" or "Website preview" (for example with Redirect pages). 
This messaging and supporting iconography doesn't have sufficient contrast to meet the accessibility standards of 3:1 for images and 4.5:1 for text.

I recommend using text-light variable (text-muted) for both the text and iconography, image should be swapped out to a 
SVG or icon font for crispness. The icon could also be removed completely alternatively, then recenter the text and think the text could also be reduced in font size and font weight (e.g. 16px and 400 weight).

![image](https://user-images.githubusercontent.com/555033/123571588-a859ad80-d81e-11eb-8ab7-cde98420ab86.png)
